### PR TITLE
added background image to header added socialization gem and follows …

### PR DESCRIPTION
**Pull request by DarthDARKFather**
…to route rb

Ready to merge, this is part of a pair programming tag team so the follows feature will be finalized later in a different pull request.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@
     end
     resources :comments, only: :create
   end
+  resources :users do
+    member do
+      post :follow
```
Comment (kenmazaika) [inline config/routes.rb]: Does this actually work?  I think you need to add work in the users controller too, unless I'm mistaken.

Comment (DarthDARKFather) [inline config/routes.rb]: All of that will be added in another pull request, I just wanted this merged because I know other people have been merging stuff and I wanted this added before any new merges, to avoid a merge conflict.

Comment (kenmazaika) [inline config/routes.rb]: Got it.  Makes sense.
